src/proxy-ui-api/src/main/resources/openapi-definition.yaml

```diff
@@ -5148,8 +5148,6 @@ components:
           description: access right given at
           example: '2018-12-15T00:00:00.001Z'
           readOnly: true
-          minLength: 24
-          maxLength: 24
```

Comment:
I removed it because it creates next exception.
```
HV000030: No validator could be found for constraint 'javax.validation.constraints.Size' validating type 'java.time.OffsetDateTime'. Check configuration for 'addServiceClientAccessRights.accessRights.items[].rightsGivenAt'
javax.validation.UnexpectedTypeException: HV000030: No validator could be found for constraint 'javax.validation.constraints.Size' validating type 'java.time.OffsetDateTime'. Check configuration for 'addServiceClientAccessRights.accessRights.items[].rightsGivenAt'
```

Reply:
Makes sense. Maybe the old generator did not even try to add the constraint annotations.